### PR TITLE
Add Load more feature

### PR DESCRIPTION
**Pull request by Sparsh1212**
Add load more functionality to Groups posts and team members.

### CODE REVIEW

File: src/components/group-post-list.js [hunk 1]
```diff
@@ -1,13 +1,24 @@
 import React from 'react'
 import { connect } from 'react-redux'
-import { Grid, Placeholder, Segment, Icon, Card } from 'semantic-ui-react'
+import {
+  Grid,
+  Placeholder,
+  Segment,
+  Icon,
+  Card,
+  Button
```
Comment (algomaster99) [inline src/components/group-post-list.js]: trailing comma

**Review by algomaster99 — CHANGES_REQUESTED** [review]
@Sparsh1212 looks good to me! Great work :tada: 
However,
- reword your commit messages as we talked
- incorporate one small change I put forward

We are good to go otherwise :smile:

**Review by pradumangoyal — APPROVED** [review]

**Comment by pradumangoyal** [comment]
@Sparsh1212 Nice work, can you implement the same in feeds also

**Comment by Sparsh1212** [comment]
@pradumangoyal  Yes! on the way for it.

**Pull request merged**